examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,39 @@
+import { ITree } from './tree.interface';
+import { PreciseProofs } from 'ew-precise-proofs-js';
+
+export class MerkleTree {
+  private tree: ITree = { leaves: [], root: undefined };
```

Comment:
can MerkleTree class contain just leaves and root fields, why it needs tree object inside merkle tree?